### PR TITLE
fix(Script/Core): Move hardcoded text to DB

### PR DESCRIPTION
**Pull request by avarishd**
Everything is in DB and it's properly linked

## Changes Proposed:
-  Same minor changes to Your Inner Turmoil to be more blizzlike
-  Added missing text to A Hero's Burden
- Still At It barrel is lootable only once, before it never despawned and you could loot it 100 times

## Issues Addressed:

- https://github.com/azerothcore/azerothcore-wotlk/issues/4165
- zone_wintergrasp -> to DB
- zone_howling_fjord -> to DB

## SOURCE:

https://youtu.be/irJrrqQpd_Y?t=98 - A Hero's Burden
https://youtu.be/RW_U8aAA8wI?t=114 - Your Inner Turmoil
## Tests Performed:

- Tested all in-game 

## How to Test the Changes:

**Preferably with non English client**

>.q a 11322 -- Alliance
.q a 11317 -- Horde
.go xyz 3027 -5088 723 571
---
>.bf timer 1 50 (If WG battle isn't within 15m)
speak with any npc that can queue you for WG
---
>.q add 12634
.npc add 28604
---
>.q add 12645
.go c id 28568
---
>.q add 12621
.go c id 27801
---
>.q add 12573
.go c id 28315
---
>.q a 12644
.go c id 28566
---
>.q a 12581
.go c id 28659
(reset if the mobs aren't chained, happens if u tp to him)

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/scripts/Northrend/zone_sholazar_basin.cpp [hunk 1]
```diff
@@ -305,6 +317,8 @@ class npc_still_at_it_trigger : public CreatureScript
 
         npc_still_at_it_triggerAI(Creature* pCreature) : NullCreatureAI(pCreature) {}
 
+        Creature* GetManus() {return ObjectAccessor::GetCreature(*me, thunderbrewGUID);}
```
Comment (Kitzunu) [inline src/server/scripts/Northrend/zone_sholazar_basin.cpp:320]: is there ever a risk of this returning null?

Comment (avarishd) [reply src/server/scripts/Northrend/zone_sholazar_basin.cpp:320]: AFAIK, no https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/scripts/Northrend/zone_sholazar_basin.cpp#L491, creature cannot die and cannot engage into any type of combat, doesn't despawn and GUID is obtained only by speaking to it (which is the only way the script begins in first place).

If I wanted to add a check, would it be something like this:
```c++
void GetManus()
{
        if (Creature* cr= ObjectAccessor::GetCreature(*me, thunderbrewGUID))
        {
                return cr;
        }
        else
        {
                Reset();
        }
}

**Comment by mpfans** [comment]
```.q a 11322 -- Alliance
.q a 11317 -- Horde
.go xyz 3027 -5088 723 571```
tested and work.

```
.bf timer 1 50 (If WG battle isn't within 15m)
speak with any npc that can queue you for WG
```
tested and work.

```
.q add 12634
.npc add 28604
```
tested and work.

```
.q add 12645
.go c id 28568
```
tested and work.

```
.q add 12621
.go c id 27801
```
tested and work.

```
.q add 12573
.go c id 28315
```
tested and work.

```
.q a 12644
.go c id 28566
```
tested and work.

```
.q a 12581
.go c id 28659
(reset if the mobs aren't chained, happens if u tp to him)
```
tested and work.

**Comment by mpfans** [comment]
Please merge it

**Comment by mpfans** [comment]
@Nyeriah

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**